buildSrc/src/main/java/com/uber/okbuck/core/model/jvm/JvmTarget.java

```diff
@@ -323,6 +328,66 @@ public static String javaVersion(JavaVersion version) {
     return optionBuilder.build();
   }
 
+  private List<String> readKotlinCompilerArguments() {
+    try {
+      // Note: this is bad juju on Gradle 5.0, which would prefer you get the provider and lazily
+      // eval.
+      // We don't differentiate between test and non-test right now because Android projects don't
+      // support test-only configuration. Non-android projects theoretically can, but let's wait for
+      // someone to ask for that support first as it's not very common.
+      Optional<KotlinCompile> kotlinCompileTask = getProject()
+          .getTasks()
+          .withType(KotlinCompile.class)
+          .stream()
```

Comment:
should we sort this to be deterministic?

Reply:
c3aab0b